Master Project/Assets/Scenes/Combat/Qi_Scripts/MonsterAction.cs

```diff
@@ -2,14 +2,127 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
+using System;
 
 public class MonsterAction : MonoBehaviour {
 
-    public enum Monster {Nessie, Cerberus, REDACTED};
-    public Animator Hitted;
+    System.Random monsterdice = new System.Random();

```

Comment:
These do follow a singular naming convention, but not Unity's standard. Just a note.